### ISSUE TITLE
Contributor

### ISSUE THREAD

**Issue by aaronjcopley** [TheresAFewConors/Sooty#27]
I appreciate the call out as a contributor for the Proofpoint script, but I can't take credit for it. I wanted to be sure that you saw my reply on Gist (https://gist.github.com/aaronjcopley/65a5198bf7b35361fdd315e786be9b9d#gistcomment-2936712) because I also noted there is a newer version published there.

Thanks!

**Comment by TheresAFewConors** [comment]
Hi Aaron, 
I actually ended up using your modified block for a long time since it seemed to be smaller and suited perfectly, its been tweaked slightly now but still functions almost identically. Hence why I contributed you instead of ProofPoint.

I saw the reply, and meant to respond but didn't get a chance, if you want I can take you off the contributors list, although i'm happy to have you on there.

-C
